### DISCUSSION TITLE
Branch name affecting visibility of 'Code' dropdown?

### DISCUSSION THREAD

**Discussion by SenneS** [community/community#60274] [Repositories]
### Select Topic Area

Bug

### Body

It seems that having a '/' in a branch's name causes GitHub to believe that it is not in the root of the repo when it actually is, thus making it not display the 'Code' dropdown button, making it harder to download a repository through the web interface.
I do not believe this is intended behaviour.
<hr>

![image](https://github.com/community/community/assets/71817687/efec632e-521a-49f3-a1bb-772fdbd9ff35)
https://github.com/SenneS/reimagined-broccoli/tree/main
<hr>

![image](https://github.com/community/community/assets/71817687/5e76e4b8-beb3-4c7d-9900-883af379d88f)
https://github.com/SenneS/reimagined-broccoli/tree/feature/main

**Comment by aagranovExtend** [comment]
I'm seeing this too with branches that have / in them.
